setup.py

```diff
@@ -1,18 +1,17 @@
 #!/usr/bin/env python3
-
-# Start ignoring PyImportSortBear as imports below may yield syntax errors
-from bears import assert_supported_version
```

Comment:
Can you explain more please? In my new code `import` are first and `from` comes after.